pennylane/transforms/optimization/undo_swaps.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Transform that eliminates the swap operators by reordering the wires."""
+# pylint: disable=too-many-branches
+from pennylane import apply
+from pennylane.transforms import qfunc_transform
+from pennylane.wires import Wires
+from pennylane.tape import stop_recording
+
+
+@qfunc_transform
+def undo_swaps(tape):
+    """Quantum function transform to remove SWAP gates. To do this it runs from right to left through the circuit changing the position of the qubits accordingly.
```

Comment:
```suggestion
    """Quantum function transform to remove SWAP gates. To do this it runs from right
    to left through the circuit changing the position of the qubits accordingly.
```
Probably want to wrap this a bit :)